protected/tests/support/BrowserSignInSteps.php

```diff
@@ -14,11 +14,22 @@ trait BrowserSignInSteps
 	public function loginToWebSiteWithSessionAndCredentialsThenAssert($email,$password,$to_assert)
 	{
 
-        $this->session->visit("http://gigadb.dev/site/login");
+        $this->session->visit("http://gigadb.dev/site/admin");
```

Comment:
I'm not sure changes are needed in this file? For example, look at `AdminFileTest.php` for how to use `BrowserSignInSteps`.